docs/content/en/docs/Concepts/DigitalShadow/index.md

```diff
@@ -315,80 +382,71 @@ microservice doing that task. It accepts specific requests, accesses the timesca
 the data in the desired format.
 
 #### Implemented functionality for digital shadow
-The following function returns all uniqueProducts for that specific asset and step_id in a specified time range. One datapoint contains one 
-childUID, all parentUID's and all available alternativeUniqueProductID's. All uniqueProductTags and 
+The following function returns all uniqueProducts for that specific asset in a specified time range. One datapoint contains one 
+childUID, AID and all parentAID's regarding the asset. All uniqueProductTags and 
 uniqueProductTagStrings (value and timestamp) for the childUID are returned to the same datapoint.
 
 `get /{customer}/{location}/{asset}/uniqueProductsWithTags`
-from `<timestamp1>` to `<timestamp2>` (in RFC 3999 Format) and for a specific `AssetID` and `step_id`.
+from `<timestamp1>` to `<timestamp2>` (in RFC 3999 Format).
 
-Example Format: 
+Example Return with two data points: 
 ```json
-[
-   row
-
-]
-
-  row:
 {
-"ValveHeadAlternativeUniqueID": str<123>,
-"TorqueScrew1": number<5.0>,
-"TorqueScrew2": number<5.0>,
-"TorqueScrew3": number<5.0>,
-"TorqueScrew4": number<5.0>,
-"gasketUID": number<5.0>,
-"childUID": number<5.0>,
-"Timestamp_ValveHeadID": number<5.0>,
-"Timestamp_GasketImageID": number<5.0>
+  "columnNames":
+  [
+    "UID",
+    "AID",
+    "TimestampBegin",
+    "TimestampEnd",
```

Comment:
todo: change names to `timestamp` and `timestampEnd`. Additionally add comment:
Instead of using timestampBegin we use timestamp because the umh-datasource for grafana will convert the column with the exact name `timestamp` to type time so grafana accepts the time series.